ckanext/dcatapit/harvesters/csw_harvester.py

```diff
@@ -33,6 +54,15 @@
         multiplicity="1",
      ))
 
+ISODocument.elements.append(
```

Comment:
You are using
**conformity-specification-title** for both identifier and title as we discussed.
So this piece of code can be removed